modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= Support for {oadp-full}
+
+.Supported versions of {oadp-short}
+[width="100%",cols="10%,12%,12%,13%,13%,20%,20%, options="header",]
+|===
+
+|Version
+|{OCP-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+|Extended Update (EUS)
+|Extended Update Support Term 2 (EUS Term 2)
+
+
+
+|1.4
+a|
+* 4.14
+* 4.15
+* 4.16
+|10 Jul 2024
+|Release of 1.5
+|Release of 1.6
+a|
+27 Jun 2026
+
+EU must be on {OCP-short} 4.16
+a|
+27 Jun 2027
+
+EUS Term 2 must be on {OCP-short} 4.16
+
+|1.3
+a|
+* 4.12
+* 4.13
+* 4.14
+* 4.15
+|29 Nov 2023
+|10 Jul 2024
+|Release of 1.5
+a|
+31 Oct 2025
+
+EUS must be on {OCP-short} 4.14
+a|
+31 Oct 2026
+
+EUS Term 2 must be on {OCP-short} 4.14
+
+7+|Previous versions of the {oadp-short} Operator which are no longer supported
+
+|1.2
+|N/A
+|14 Jun 2023
+|29 Nov 2023
+|10 Jul 2024
+|
+|
+
+|1.1
+|N/A
+|01 Sep 2022
+|14 Jun 2023
+|29 Nov 2023
+|
+|
+
+|1.0
+|N/A
+|09 Feb 2022
+|01 Sep 2022
+|14 Jun 2023
```

Comment:
Could this whole part be a new table? With `.Previous versions of the {oadp-short} Operator which are no longer supported` being the new title?

I think it would look a little cleaner, and also since it's currently in a table titled `.Supported versions of {oadp-short}`, which is a little strange even with the "these ones are unsupported" caveat in place.